beacon-chain/slasher/helpers_test.go

```diff
@@ -0,0 +1,264 @@
+package slasher
```

Comment:
This is simply moved here from detect_attestations_test.go, no new logic nor tests